src/Pwa/PageLoader/ProductPageLoader.php

```diff
@@ -88,10 +84,11 @@ public function load(PageLoaderContext $pageLoaderContext): ProductPageResult
             new EqualsFilter('active', 1)
         );
 
+        $criteria->addAssociation('seoUrls');
```

Comment:
I think we don't need to specify that association, as it is loaded whenever we include the `sw-include-seo-urls` header for our request. We used to have a small bug regarding that resolving for sales channel entities which should have been fixed in 6.3.* and will be available very soon.